src/lib.rs

```diff
@@ -1,9 +1,11 @@
-use std::collections;
-use std::env;
-use std::error::Error;
-use std::io;
-use std::io::BufRead;
-use std::path;
+use std::{
+    collections::{HashMap, HashSet},
```

Comment:
Is it common in Rust to strip out the module from the namespace and reference down to only the type?